libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -711,6 +711,9 @@ fn create_heartbeat() -> Result<Vec<u8>, ConnectionManagerError> {
 #[cfg(test)]
 pub mod tests {
     use super::*;
+
```

Comment:
Typo in commit message "can be remove by" -> "can be removed by"

Reply:
Fixed (the typo was in the test comment too)